osu.Game.Rulesets.Osu/Mods/OsuModTarget.cs

```diff
@@ -278,7 +284,7 @@ private void randomizeCirclePos(IReadOnlyList<OsuHitObject> hitObjects)
         {
             if (hitObjects.Count == 0) return;
 
-            float nextSingle(float max = 1f) => (float)(rng.NextDouble() * max);
+            float nextSingle(float max = 1f) => (float)(rng.AsNonNull().NextDouble() * max);
```

Comment:
This could be avoided by not storing `rng` as a field, but pass it from `ApplyToBeatmap` as an argument to this method instead. Same applies to `originalHitObjects`.

Reply:
fixed 🙆🏼‍♂️